### PR TITLE
Feat/update readme

### PR DESCRIPTION
**Pull request by megstanley**
Update with links to dataset and models. Please check that these are functioning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ The code for the Molecule Attention Transformer baseline is added as a submodule
 
 ## Data
 
-The actual dataset is stored in `dataset/`, split into `train`, `valid` and `test` folders.
+The dataset is available as a download, [FS-Mol Data](https://figshare.com/ndownloader/files/31345321), split into `train`, `valid` and `test` folders. Additionally, we specify which tasks are to be used with the file `dataset/fsmol-0.1.json`, a default list of tasks for each data fold. We note that the complete dataset contains many more tasks. Should use of all possible training tasks available be desired, the training script argument `--task_list_file` `dataset/entire_train_set.json` should be used. The task lists will be used to version FS-Mol in future iterations as more data becomes available via ChEMBL.
```
Comment (kmaziarz) [inline README.md:35]: The link works for me, and downloads a valid `*.tar` file with the directory structure as described 🙂

But, I'm assuming it should be
```suggestion
The dataset is available as a download, [FS-Mol Data](https://figshare.com/ndownloader/files/31345321), split into `train`, `valid` and `test` folders. Additionally, we specify which tasks are to be used with the file `datasets/fsmol-0.1.json`, a default list of tasks for each data fold. We note that the complete dataset contains many more tasks. Should use of all possible training tasks available be desired, the training script argument `--task_list_file` `datasets/entire_train_set.json` should be used. The task lists will be used to version FS-Mol in future iterations as more data becomes available via ChEMBL.
```

By the way, is "available as a download" a valid expression?

File: README.md [hunk 2]
```diff
@@ -144,13 +145,13 @@ python fs_mol/protonet_test.py /path/to/pn-checkpoint /path/to/data
...
-| GNN-MT     | 10-layer GNN. PNA message passing                                         | [Multitask_best_models](https://figshare.com/ndownloader/files/30560865)             |
-| PN         | 10-layer GGN, PNA message passing. ECFP+GNN, Mahalonobis distance metric  | [PN-Support64_best_validation.pt](https://figshare.com/ndownloader/files/30560862)   |
+| Model Name | Description                                                              | Checkpoint File                                                                       |
+| ---------- | ------------------------------------------------------------------------ | ------------------------------------------------------------------------------------- |
+| GNN-MAML   | Support set size 16. 8-layer GNN. Edge MLP message passing.              | [MAML-Support16_best_validation.pkl](https://figshare.com/ndownloader/files/31346701) |
+| GNN-MT     | 10-layer GNN. PNA message passing                                        | [Multitask_best_models](https://figshare.com/ndownloader/files/31338334)              |
```
Comment (kmaziarz) [inline README.md:153]:
```suggestion
| GNN-MT     | 10-layer GNN. PNA message passing                                        | [multitask_best_model.pt](https://figshare.com/ndownloader/files/31338334)              |
```
(to match the name of the file that actually gets downloaded)

**Review by kmaziarz — APPROVED** [review]
Left a few nits, but LGTM overall 🙂

File: README.md [hunk 3]
```diff
@@ -144,13 +145,13 @@ python fs_mol/protonet_test.py /path/to/pn-checkpoint /path/to/data
 
 ## Available Model Checkpoints
 
-We provide pre-trained models for `GNN-MAML`, `GNN-MT` and `PN`, these are downloadable from the links to [figshare](https://figshare.com/articles/dataset/FSMol_Baseline_Models/16511568).
+We provide pre-trained models for `GNN-MAML`, `GNN-MT` and `PN`, these are downloadable from the links to [figshare](https://figshare.com/account/home#/projects/125797).
```
[the remark this answers is not in the document]
Comment (megstanley) [reply README.md:148]: Oh that's weird. It's meant to give a link to the project. Let me fix that.

**Pull request merged**